### DISCUSSION TITLE
Brew considers the macupdater cask outdated because of a version that cannot run on my macOS version

### DISCUSSION THREAD

**Discussion by rgoldberg** [Homebrew/discussions#5448] [Casks] [answered]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.3.6-36-g14d452f
ORIGIN: https://github.com/Homebrew/brew.git
HEAD: 14d452fcde0c985e091d359fbc87d69e5c730048
Last commit: 10 hours ago
Core tap HEAD: 50acb329b0408ec88c1ee57deea0cdcc1aa1bf53
Core tap last commit: 36 minutes ago
Core tap JSON: 20 Jun 22:49 UTC
Core cask tap HEAD: 84a66a4e79e90b5cc32a3829e6efe4e880b076e4
Core cask tap last commit: 69 minutes ago
Core cask tap JSON: 20 Jun 22:57 UTC
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_DISPLAY: /private/tmp/com.apple.launchd.U0RlI8f24e/org.xquartz:0
HOMEBREW_GITHUB_API_TOKEN: set
HOMEBREW_MAKE_JOBS: 4
HOMEBREW_NO_ANALYTICS: set
HOMEBREW_SORBET_RUNTIME: set
Homebrew Ruby: 3.3.3 => /usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/3.3.3/bin/ruby
CPU: quad-core 64-bit haswell
Clang: 14.0.0 build 1400
Git: 2.45.2 => /usr/local/bin/git
Curl: 8.4.0 => /usr/bin/curl
macOS: 12.7.5-x86_64
CLT: 14.2.0.0.1.1668646533
Xcode: 14.2
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

Brew considers the `macupdater` cask outdated because of a version (3.3.1) that cannot run on my macOS version (Monterey 12.7.5)

MacUpdater 3+ only works on Ventura or newer.

MacUpdater 2.x (2.3.15 is current) works on Monterey.

IIRC, the following command previously output nothing (maybe a week or so ago), but it currently indicates that `macupdater` is outdated.

```
$ brew outdated --cask --greedy
macupdater (2.3.15) != 3.3.1
```

How can I ensure that brew doesn't consider versions that cannot run on Monterey?

This might have started around the time when brew installed its own ruby install.

### Relevant casks

macupdater

**Comment by EricFromCanada** [comment, marked as answer]
Thanks for the report. This should be fixed by https://github.com/Homebrew/brew/pull/17540.
